lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -1297,4 +1312,73 @@ class StandardAccountTest {
             assertEquals(1, messages[1].order)
         }
     }
+
+    @Nested
+    inner class OutsideExecutionTest {
+        private val randomAddress = Felt.fromHex("0x655b2696f187daec1952ef59befbd875add757748d6c844fb273439cd0d82f9")
+
+        @Test
+        fun `executes transfers from another specified account`() {
+            val call1 = transferCall(100)
+            val call2 = transferCall(200)
+            val balanceBefore = getBalance(account.address)
+            val now = Instant.now()
+            val outsideCall = account.createExecuteFromOutsideV2Call(
+                caller = secondAccount.address,
+                executeAfter = Felt(now.minusSeconds(10).epochSecond),
+                executeBefore = Felt(now.plusSeconds(10).epochSecond),
+                calls = listOf(call1, call2),
+                nonce = Felt(Random.nextLong().absoluteValue),
+            )
+
+            secondAccount.executeV3(outsideCall).send()
+
+            val balanceAfter = getBalance(account.address)
+            val diff = balanceBefore - balanceAfter
+            assertEquals(diff, 300.toBigInteger())
+        }
+
+        @Test
+        fun `executes transfers from another any account`() {
+            val call1 = transferCall(100)
+            val call2 = transferCall(200)
+            val balanceBefore = getBalance(account.address)
+            val now = Instant.now()
+            val outsideCall = account.createExecuteFromOutsideV2Call(
+                caller = Felt.fromShortString("ANY_CALLER"),
+                executeAfter = Felt(now.minusSeconds(10).epochSecond),
+                executeBefore = Felt(now.plusSeconds(10).epochSecond),
+                calls = listOf(call1, call2),
+                nonce = Felt(Random.nextLong().absoluteValue),
+            )
+
+            secondAccount.executeV3(outsideCall).send()
+
+            val balanceAfter = getBalance(account.address)
+            val diff = balanceBefore - balanceAfter
+            assertEquals(diff, 300.toBigInteger())
+        }
+
+        private fun getBalance(address: Felt): BigInteger {
+            return devnetClient.provider.callContract(
+                Call(
+                    contractAddress = DevnetClient.ethErc20ContractAddress,
+                    entrypoint = "balance_of",
+                    calldata = listOf(address),
+                ),
+            ).send()[0].value
+        }
+
+        private fun transferCall(amount: Int): Call {
```

Comment:
(same for occurrences above):
```suggestion
        private fun createTransferCall(amount: Int): Call {
```